### DISCUSSION TITLE
Orcaslicer and GX files

### DISCUSSION THREAD

**Discussion by urself25** [ronoaldo/FlashforgeFinderIntegration#27] [General]
Hi,

I know Orcaslicer doesn't have a marketplace or (to my knowledge) the ability to add extensions. But have you ever thought of trying to make your scripts work with Orcaslicer?

Thanks.

**Comment by ronoaldo** [comment]
Hi! I did. I failed to add the printer to OrcaSlicer as a custom one so I don't even know if Orca can generate a valid g-code to the Finder v2.

**Reply by urself25** [reply]
Thanks for the quick response. 

Have you posted those files anywhere. I am wondering if I could start from those attempts to try on my own. I'm a self learner and with way less knowledge and experience as you do, but I would like to take a shot at it also.

Thanks.

**Reply by ronoaldo** [reply]
You mean, the files for adding the printer to Orca? No, I have not reached anyewhere besides trying to create a custom printer. It draws the bed off center and I am concerned it would just create a bad format g-code. In cura, we can have something like a center coordinate (i guess the finder firmware may require this, I could be wrong), but if you manage to build a profile that can print a sample cube/benchy I can try to port the .gx header to be a standalone script that one can add as a post-processing step.

**Comment by urself25** [comment]
Hi Ronoaldo, 

I made a fork of your GxWriter to make it work as a standalone conversion tool which also works with Orcaslicer. I'm no programmer/developer. I used ChatGPT to remove Cura-Specific dependencies to make you script work. I still have issues with the bitmap integration. Orca renders a png thumbnail, and I'm trying to integrate a conversion tool from PNG to BMP into the script. 

If you want to take a look at it: https://github.com/urself25/GxWriter2
